### ISSUE TITLE
Search results order within the url parameter

### ISSUE THREAD

**Issue by sizzlingkenny** [piratebayproxy/UnblockedPiratebayClean#27]
is there a way to sort search results by adding parameter to the url like 

`/search.php?q=crazy+bird&cat=0&uploaded=desc`

i know the order feature is already in the script by using javascript but i want this feature to work through the url.

**Comment by piratebayproxy** [comment]
I do not think TPB API has such functionality.

You could add this functionality by creating JS on search.php page to sort by how you desire on page load. This method is less laborious.

Other option is by modifying api.php to accept 'uploaded' GET parameter and then sorting json result however you desire (maybe there exists a library for sorting json based on values)

**Comment by sizzlingkenny** [comment]
unfortunately i'm not good with javascript or json but i do know some php anyways thanks for the suggestion

**Comment by erikewp** [comment]
@piratebayproxy Actually, the sort function is handled by the main.js file. Some tweaks in that file can change how the sort is handled on page load. The default TPB sorting is pretty bad so it could be something worth improving in the future. The old TPB layout was much better in this regard.

**Comment by sizzlingkenny** [comment]
yeah exactly my point.. if only sort should work by `time desc` will be more than enough because right now its working as `seed desc` and shows unwanted and older results and that's a bad thing.
